core/src/test/java/io/customer/sdk/core/di/DiGraphTest.kt

```diff
@@ -231,3 +232,6 @@ class DiGraphTest : BaseUnitTest() {
         diGraph.getOrNull<Float>().shouldBeNull()
     }
 }
+
+// Custom data class to test dependency injection with the same class name.
+private data class Pair(val value: Int)
```

Comment:
why is the naming `Pair`?, should we name it something that more suitable here and then wont have to use `kotlin.Pair` as well to avoid conflict

Reply:
That's exactly the test is, it should not fail if class names are same 😅